module/zcommon/zfs_fletcher.c

```diff
@@ -133,11 +133,11 @@
  * than sha-256, and slower than 'off', which doesn't touch the data at all.
  */
 
+#include <sys/simd.h>
 #include <sys/types.h>
```

Comment:
Again, I doubt simd.h should be before types.h.  Wouldn't before spa.h be enough?

Reply:
Yes, that is enough. It has to be processed before vnode.h (which is called by zfs_context.h)